x-pack/plugins/ml/public/application/components/annotations/annotations_table/annotations_table.js

```diff
@@ -126,6 +142,57 @@ export class AnnotationsTable extends Component {
     return mlJobService.getJob(jobId);
   }
 
+  handleSearchChange({ query, queryText, error }) {
```

Comment:
this function definitely deserves a unit test 🙂

Reply:
I ended up using a different aoproach and removing this function here https://github.com/elastic/kibana/pull/70198/commits/5b6249d33c3f2b1b59fecfbe2baf899f1e32a002